common/changes/@typespec/compiler/remove-emitter-not-found-check_2023-05-23-15-33.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "Remove `emitter-not-found` validation",
```

Comment:
Given that it will still validate if you give a bogus emitter name, we should probably rephrase this. "Fix issue where emitters that were not invoked are claimed to not exist." or something like that.